tests/ci_build/Dockerfile.jvm_cross

```diff
@@ -20,9 +20,10 @@ RUN \
     tar xvf apache-maven-3.6.1-bin.tar.gz -C /opt && \
     ln -s /opt/apache-maven-3.6.1/ /opt/maven && \
     # Spark
-    wget https://archive.apache.org/dist/spark/spark-$SPARK_VERSION/spark-$SPARK_VERSION-bin-hadoop2.7.tgz && \
-    tar xvf spark-$SPARK_VERSION-bin-hadoop2.7.tgz -C /opt && \
-    ln -s /opt/spark-$SPARK_VERSION-bin-hadoop2.7 /opt/spark
+    # This should be: wget https://archive.apache.org/dist/spark/spark-$SPARK_VERSION/spark-$SPARK_VERSION-bin-scala2.12-hadoop2.7.tgz && \
```

Comment:
@hcho3 shall we put this file to the image of our CI workers?

Reply:
@CodingCat Apache doesn't distribute artifacts with 2.12?

Reply:
2.12 is in maven central, but no downloadable artifact

Reply:
@CodingCat Got it, we can put it in a S3 bucket and have Jenkins CI workers access it

Reply:
@hcho3 would you mind putting this file to a place which is under your control instead of using @koertkuipers's personal account? after that I can merge it with some changes

Reply:
ping @hcho3

Reply:
@CodingCat Done

Reply:
thanks @hcho3